admin/howto/environment.md

```diff
@@ -21,17 +21,62 @@ It is configured with the following:
   - [RStudio](https://rstudio.com/)
 - An Ubuntu 20.04 base image, with common utility packages installed.
 
+(environment:default-interface)=
+### Default user interfaces
+
+The 2i2c hubs offer the following user interfaces by default:
+
+#### Jupyter Notebook (Classic)
+
+The original single-document interface for creating Jupyter Notebooks.
+
+#### JupyterLab
+
+
+```{figure} ../../images/jupyterlab.png
+:alt: JupyterLab layout
+```
+
+[JupyterLab](https://github.com/jupyterlab/jupyterlab) is a more modern version of the classic Jupyter notebook from
+the Jupyter project. It is more customizable and better supports advanced use cases - particularly around [dask](https://dask.org). Many
+research organizations use this.
+
+#### RStudio
+
+```{figure} ../../images/rstudio.png
+:alt: RStudio
+```
+
+[RStudio](https://rstudio.com) is an IDE for R, created by the RStudio company.
+
+## Choose the user interface
+
+There are a three main interfaces available on the 2i2c JupyterHubs. You may choose from each of these by altering the URL of your session. For example, here is the general structure of a URL for your personal 2i2c JupyterHub session:
+
+```
+https://<your-hub>.pilot.2i2c.cloud/user/<your-username>/<your-interface>
+```
+
+You can replace the contents of `<your-interface>` to be one of the following:
+
+- **JupyterLab**: `/lab`
+- **Jupyter Notebook**: `/tree`
+- **RStudio**: `/rstudio`
+
+Note that a Hub Administrator can also configure the **default** interface that users see. In addition, you can configure the interface that **nbgitpuller links** point to, see [](content:nbgitpuller) for information about nbgitpuller links.
```

Comment:
I think it would be useful to link to the info on how to change the default interface here.

Reply:
where is that information? I looked around in this repo and didn't see where would be the natural place to link to